index.rst

```diff
@@ -58,24 +58,216 @@ We describe the design and implementation of the LSST Alert Distribution System,
 Alert Serialization
 ===================
 
+Alerts are packaged using Apache Avro.
+Avro is a framework for data serialization in a compact binary format.
+It has been used at scale in both industry and science, and it is the
+recommended format for data streamed with Apache Kafka.
+Avro is more structured in format than JSON or XML, the currently used
+format of VOEvent 2.0.
+Libraries for reading and writing Avro are available in many languages,
+including Python.
+
+To create an Avro alert packet, data is serialized using a JSON schema,
+which defines the data types for each field.
+Strict adherence to the schema ensures that data will be correctly
+interpreted upon receipt.
+Alerts in Avro format can be shipped with or without a schema embedded,
+allowing them to be more light weight.
```

Comment:
I'd describe the practical tradeoff in excluding the schema, which is that you need a way to retrieve the schema appropriate to your message.